draft-ietf-quic-qpack.md

```diff
@@ -627,14 +632,14 @@ type HTTP_WRONG_STREAM_COUNT. These streams MUST NOT be closed. Closure of
 either unidirectional stream type MUST be treated as a connection error of type
 HTTP_CLOSED_CRITICAL_STREAM.
 
-An endpoint MAY avoid creating its own encoder stream if it's not going to be
-used (for example if the endpoint doesn't wish to use the dynamic table, or if
```

Comment:
Each endpoint has two contexts and each context has a dynamic table, so it's better to clarify which one.